internal/images/images.go

```diff
@@ -29,13 +22,32 @@ func init() {
 
 func GetLatestTag(
 	repoURL string,
-	updateStrategy ImageUpdateStrategy,
+	updateStrategy kargoapi.ImageUpdateStrategy,
 	semverConstraint string,
 	allowTags string,
 	ignoreTags []string,
 	platform string,
 	creds *Credentials,
 ) (string, error) {
+	// TODO: Be on the lookout for an Image Updater release >= 0.13.0.
+	//
+	// Image Updater is changing the name of two update strategies to avoid
+	// confusion caused by their older names.
+	//
+	// This change can be seen here:
+	//   https://github.com/argoproj-labs/argocd-image-updater/pull/456
+	//
+	// This change has not yet made its way into an Image Updater release, but we
+	// want to support these new strategy names NOW to avoid the same confusion.
```

Comment:
Not going to gate the PR, but we could import image updater from tip if we want the changes now.

Longer term, we need a way to untangle ourselves from argo-cd, gitops-engine, image-updater, even if it means copying and pasting the libraries that we want. But that's a whole other effort.

Reply:
> Not going to gate the PR, but we could import image updater from tip if we want the changes now.

That's not a bad idea, especially since there's [another change](https://github.com/argoproj-labs/argocd-image-updater/pull/559) that's been merged into image updater's main branch that we currently have [a workaround for](https://github.com/akuity/kargo/blob/main/internal/images/semver.go#L5-L16) while waiting for it to be released.

Reply:
Using the latest from their `master` branch now, which enabled us to drop two workarounds.

> Longer term, we need a way to untangle ourselves from argo-cd, gitops-engine, image-updater, even if it means copying and pasting the libraries that we want. But that's a whole other effort.

💯